### PR TITLE
Update Project Metadata and Naming in Node

### PR DESCRIPTION
**Pull request by asiniscalchi**
## PR Type:
Refactoring

___
## PR Description:
This PR includes changes to update the project metadata and naming in the node. The main changes include:
- Renaming from "Frontier Parachain Collator Template" to "LAOS Parachain Node"
- Updating the support URL to point to the new project repository
- Changing the copyright start year from 2020 to 2023
- Updating the project metadata in the Cargo.toml file

___
## PR Main Files Walkthrough:
<details> <summary>files:</summary>

- `ownership-chain/node/src/command.rs`: Updated the implementation name, description, support URL, and copyright start year for both Cli and RelayChainCli.
- `ownership-chain/node/Cargo.toml`: Updated the project metadata including the homepage and repository URLs.
</details>

### CODE REVIEW

File: ownership-chain/node/Cargo.toml [hunk 1]
```diff
@@ -1,12 +1,8 @@
 [package]
 name = "laos-ownership"
 version = "0.1.0"
-authors = ["Anonymous"]
-description = "A new Cumulus FRAME-based Substrate Node, ready for hacking together a parachain."
-license = "Unlicense"
-homepage = "https://substrate.io"
-repository = "https://github.com/paritytech/cumulus/"
 edition = "2021"
+authors = { workspace = true }
```
Comment (magecnion) [inline ownership-chain/node/Cargo.toml:5]: maybe a typo?

**Review by magecnion — CHANGES_REQUESTED** [review]

File: ownership-chain/node/Cargo.toml [hunk 2]
```diff
@@ -1,12 +1,8 @@
 [package]
 name = "laos-ownership"
 version = "0.1.0"
-authors = ["Anonymous"]
-description = "A new Cumulus FRAME-based Substrate Node, ready for hacking together a parachain."
-license = "Unlicense"
-homepage = "https://substrate.io"
-repository = "https://github.com/paritytech/cumulus/"
 edition = "2021"
```
Comment (magecnion) [inline ownership-chain/node/Cargo.toml:4]: 2023?

**Review by magecnion — CHANGES_REQUESTED** [review]

File: ownership-chain/node/Cargo.toml (hunk 2 above)
Comment (asiniscalchi) [reply ownership-chain/node/Cargo.toml:4]: it's the edition of rust

File: ownership-chain/node/Cargo.toml (hunk 1 above)
Comment (asiniscalchi) [reply ownership-chain/node/Cargo.toml:5]: no no it's correct by "Freeverse"
``` 
2023-11-20 18:20:12 LAOS Parachain Node    
2023-11-20 18:20:12 ✌️  version 0.1.0-126c9b47bcf    
2023-11-20 18:20:12 ❤️  by Freeverse, 2023-2023    
2023-11-20 18:20:12 📋 Chain specification: klaos network    
2023-11-20 18:20:12 🏷  Node name: recondite-tongue-0177    
```

**Review by magecnion — DISMISSED** [review]

File: ownership-chain/node/src/command.rs [hunk 3]
```diff
@@ -93,7 +93,7 @@ impl SubstrateCli for RelayChainCli {
 
 	fn description() -> String {
 		format!(
-			"Frontier Parachain Collator Template\n\nThe command-line arguments provided first will be \
+			"LAOS Parachain Node\n\nThe command-line arguments provided first will be \
 		passed to the parachain node, while the arguments provided after -- will be passed \
 		to the relay chain node.\n\n\
 		{} <parachain-args> -- <relay-chain-args>",
```
Comment (dastanfv) [inline ownership-chain/node/src/command.rs:99]: line 109 still has `parachain-template` link

Comment (asiniscalchi) [reply ownership-chain/node/src/command.rs:99]: fixed

**Review by dastanfv — APPROVED** [review]

**Pull request merged**